docs/BATS.md

```diff
@@ -4,11 +4,11 @@ This document is relevant starting from [v0.11.52](https://github.com/redkubes/o
 
 ## Where can I find BATS?
 
-The otomi/tools:1.4.10 includes the [bats framework](https://github.com/bats-core/bats-core), including libraries [bats-assert](https://github.com/ztombol/bats-assert), [bats-file](https://github.com/ztombol/bats-support) and [bats-support](https://github.com/ztombol/bats-support). These links include relevant documentation, such as syntax, which won't be discussed on this page.
+The otomi/tools:1.4.10 or newer includes the [bats framework](https://github.com/bats-core/bats-core), including libraries [bats-assert](https://github.com/ztombol/bats-assert), [bats-file](https://github.com/ztombol/bats-support) and [bats-support](https://github.com/ztombol/bats-support). These links include relevant documentation, such as syntax, which won't be discussed on this page.
 
 Example of calling the binary:
 
-`docker run --rm otomi/tools:1.4.10 bats bin/tests`
+`docker build --target test .`
```

Comment:
please don't change this...it will also run the other tests, including the one in the parent image

Reply:
The `old` line doesn't work
```
marc@Marcs-MBP otomi-core % docker run --rm otomi/tools:1.4.10 bats bin/tests
/usr/local/libexec/bats-core/bats: line 59: cd: bin: No such file or directory
```

Reply:
The otomi/tools:1.4.10 does not have bats binary
Moreover otomi/tools does not contain bin/test directory. It is included in otomi-core,
Should be:
```
docker run --rm otomi/core:latest bats bin/tests/bootstrap.bats 
```

Reply:
`otomi/tools` has the bats binary, but not the tests nor the code that should be tested. So we do need `otomi/core` to run the bats test